packages/firestore/src/util/proto_byte_string.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ProtoByteString } from '../core/types';
+import { PlatformSupport } from '../platform/platform';
+import { Blob } from '../api/blob';
+
+/**
+ * Returns the representation of an empty "proto" byte string for the
```

Comment:
This comment is inaccurate. This just returns an empty blob. There's nothing platform specific about it.